templates/flutter/lib/exception.dart.twig

```diff
@@ -0,0 +1,10 @@
+part of {{ language.params.packageName }};
+
+class {{spec.title | caseUcfirst}}Exception implements Exception {
+  final String message;
+  final int code;
+  final dynamic response;
```

Comment:
Can we maybe have a specific type here? 🤔 
`String` maybe? 
`Map<String, dynamic>` is fine as well, but as a mobile developer who is coming from strongly typed languages like kotlin, java or swift I like having types 👍

Reply:
If the response can be different in some cases it's fine to have `dynamic` but then I would add a little comment to it. In our case, we are casting to some type.

Reply:
I believe the response can be Json or can be String as well. @eldadfux do you agree?

Reply:
Yeh It might be tricky because sometimes a proxy can throw the error and we can't be sure of the format.